Clover/app/src/main/java/org/floens/chan/ui/view/MultiImageView.java

```diff
@@ -443,6 +446,10 @@ public void onEnd() {
         });
     }
 
+    private void setOther(String fileUrl) {
+        Toast.makeText(getContext(), R.string.file_not_viewable, Toast.LENGTH_LONG).show();
```

Comment:
Maybe we can show this as text in the view itself, instead of a toast. If that's too much trouble leave it for another time.

Reply:
I'm not opposed to that, i fact i think this is better. I'll try to find an example i could use to realize that, otherwise you're free to go ahead and change it to your liking.